tools/tpcds-tools/conf/opt/opt_sf1.sql

```diff
@@ -1 +1,2 @@
 set global runtime_filter_wait_time_ms=1000;
+set global enable_nereids_rules='ELIMINATE_GROUP_BY_KEY';
```

Comment:
it is danger to set global here, could we add a post process to set them back to original values?